src/actions/content.js

```diff
@@ -314,7 +314,7 @@ function* addContent({ payload: { content } }) {
     yield put(showLoading());
     yield call(api, 'node:add', { parameters: { node: content } });
   } catch (error) {
-    yield put(setMessage(error.toString()));
+    yield put(setMessage(error.toString(), MESSAGE_SEVERITY_ERROR));
```

Comment:
Just in general, I'm wondering: Could we just introduce a ```setErrorMessage``` which makes this way easier to read?

Reply:
Sounds like a good suggestion to me!

Reply:
Created new issue for this: https://github.com/jsdrupal/drupal-admin-ui/issues/375